packages/kcmsf/src/hooks/useInterval.ts

```diff
@@ -1,14 +1,20 @@
 import { useEffect, useRef } from "react";
 
-export const useInterval = (onUpdate: OnUpdate, ms: number) => {
+export const useInterval = (
+  onUpdate: OnUpdate,
+  ms: number,
+  active: boolean = true
+) => {
```

Comment:
オプションは
```suggestion
export const useInterval = (
  onUpdate: OnUpdate,
  ms: number,
  option: Option = {
    active: true
  }
) => {
```
のようにしてください。